Doc/Tutorial/chapter_quick_start.tex

```diff
@@ -168,7 +168,7 @@ \section{Connecting with biological databases}
 
 \section{What to do next}
 
-Now that you've made it this far, you hopefully have a good understanding of the basics of Biopython and are ready to start using it for doing useful work. The best thing to do now is finish reading this tutorial, and then if you want start snooping around in the source code, and looking at the automatically generated documentation.
+Now that you've made it this far, you hopefully have a good understanding of the basics of Biopython, and you are ready to start using it for doing useful work. The best thing to do now is finish reading this tutorial, and then if you want start snooping around in the source code, and looking at the automatically generated documentation.
```

Comment:
To me the new version makes it a statement "you are ready", while the original was an expression of hope only: "..., you hopefully have a good understanding ... and are ready to start ...". Either way, it is still such an overly long sentence, that I think it would benefit from a larger rephrasing. Do you have any thoughts on how to do that?